python312Packages.mkdocs-simple-blog: init at 0.2.0

Guelakais opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [x] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

jpds committed:
nixos/kafka: Moved existing test to subdirectory

jpds committed:
nixos/kafka: Added cluster testcase

jpds committed:
nixos/kafka: Added mirrormaker testcase

kuruczgy committed:
fex: minimize set of cloned submodules

This change also removes the submodules that contained prebuilt (test)
binaries.

kuruczgy committed:
fex: fix and enable tests

This will still fail on Apple Silicon devices using 16K page size
kernels, but such systems are not supported by upstream, so there is
nothing we can do about that.

nagy committed:
kubernetes-helm: make completion generation platform-dependant

nagy committed:
kubernetes-helm: use `finalAttrs` pattern

SuperSandro2000 committed:
guvcview: cleanup superfluous code

polyfloyd committed:
espup: Set passthru.updateScript

r-ryantm committed:
mattermostLatest: 10.6.0 -> 10.6.1

r-ryantm committed:
gemmi: 0.7.0 -> 0.7.1

r-ryantm committed:
janet: 1.37.1 -> 1.38.0

r-ryantm committed:
netmaker: 0.30.0 -> 0.90.0

r-ryantm committed:
rmg-wayland: 0.7.7 -> 0.7.8

marcin-serwin committed:
signal-desktop-source: add update script

r-ryantm committed:
klipper: 0.12.0-unstable-2025-03-12 -> 0.12.0-unstable-2025-03-25

rorosen committed:
rke2: use `finalAttrs` to avoid `let` and `rec` bindings

r-ryantm committed:
dnscrypt-proxy: 2.1.7 -> 2.1.8

r-ryantm committed:
sysdig-cli-scanner: 1.21.0 -> 1.22.0

r-ryantm committed:
liblscp: 1.0.0 -> 1.0.1

r-ryantm committed:
lima: 1.0.6 -> 1.0.7

dotlambda committed:
python313Packages.syrupy: 4.9.0 -> 4.9.1

Diff: https://github.com/syrupy-project/syrupy/compare/refs/tags/v4.9.0...v4.9.1

Changelog: https://github.com/syrupy-project/syrupy/blob/v4.9.1/CHANGELOG.md

r-ryantm committed:
pipe-viewer: 0.5.4 -> 0.5.5

r-ryantm committed:
astal.io: 0-unstable-2025-03-17 -> 0-unstable-2025-03-21

Commit:
dozenal: drop

r-ryantm committed:
cockpit: 331 -> 336.2

r-ryantm committed:
apt: 2.9.33 -> 2.9.35

r-ryantm committed:
magic-vlsi: 8.3.522 -> 8.3.524

r-ryantm committed:
libdatovka: 0.7.0 -> 0.7.1

r-ryantm committed:
converseen: 0.12.2.5 -> 0.13.0.1

r-ryantm committed:
python312Packages.nodriver: 0.41 -> 0.43

r-ryantm committed:
python312Packages.jianpu-ly: 1.842 -> 1.844

r-ryantm committed:
linuxPackages.oci-seccomp-bpf-hook: 1.2.10 -> 1.2.11

FliegendeWurst committed:
yarn-berry: add meta.changelog

r-ryantm committed:
python312Packages.radish-bdd: 0.18.1 -> 0.18.2

autra committed:
plan-exporter: add update script

jfly committed:
python3.pkgs.pcbnewtransition: 0.5.0 -> 0.5.2

It looks like they've started publishing sdists with lowercase filenames
(`pcbnewtransition-X.Y.Z.tar.gz` rather than
`pcbnewTransition-X.Y.Z.tar.gz`).

jfly committed:
kikit: 1.7.1 -> 1.7.2

changelog: https://github.com/yaqwsx/KiKit/releases/tag/v1.7.2

This brings in support for KiCAD v9:
https://github.com/yaqwsx/KiKit/releases/tag/v1.7.2, as [we upgraded to
KiCAD v9 recently](https://github.com/NixOS/nixpkgs/pull/384150).

illdefined committed:
eza: use finalAttrs pattern

illdefined committed:
eza: remove use of with lib

Commit:
sunshine: fix /bin/sleep in systemd service

sunshine's systemd service has hardcoded '/bin/sleep'. This fixes it by
replacing '/bin/sleep' with the full nix path to sleep from coretuils.

Resolves Issue #395748.

jopejoe1 committed:
nixops_unstablePlugins: put removed packages behind allowAliases

drakon64 committed:
cosmic-comp: remove useXWayland option

marcin-serwin committed:
signal-desktop-source: 7.48.0 -> 7.49.0

commiterate committed:
gitlab-runner: Add main program, version check hook, and Nix update script. Fix Darwin builds.

blakesmith committed:
nsq: Add blakesmith as maintainer

illdefined committed:
eza: use tag attribute for fetchFromGitHub

r-ryantm committed:
theforceengine: 1.22.200 -> 1.22.300

r-ryantm committed:
qlog: 0.42.2 -> 0.43.0

r-ryantm committed:
lxde.lxtask: 0.1.11 -> 0.1.12

jcollie committed:
figlet: modernize package

1. Add `meta.mainProgram`.
2. Use `finalAttrs` instead of `rec`.
3. Update hashes to the new style.

RossComputerGuy committed:
selinux-refpolicy: init 2.20250213

guylamar2006 committed:
ansible: Migrate to by-name

lovelymono committed:
cargo-deny: update description

Previous description was from cargo-about. Updated the description to
reflect the upstream.

Guelakais committed:
python312Packages.mkdocs-simple-blog: init at 0.2.0

Guelakais committed:
python312Packages.mkdocs-simple-blog: init at 0.2.0

roberth committed:
nixVersions.nix_2_28: init at 2.28.1pre

This includes an important fix that's not in a release yet, hence the
"pre" version.

Addendum from Ericson2314: We've discussed for 25.05 going with the
"monolithic build" for the main `nix_2_28`, but we want to land this
right away to unblock preparing PRs to bump versions downstream.

r-ryantm committed:
ansible-navigator: 25.1.0 -> 25.4.0

r-ryantm committed:
terraform-providers.temporalcloud: 0.6.1 -> 0.7.0

r-ryantm committed:
rust-analyzer-unwrapped: 2025-03-24 -> 2025-03-31

r-ryantm committed:
terraform-providers.akamai: 7.0.0 -> 7.1.0

PuercoPop committed:
maintainers: Add PuercoPop

PuercoPop committed:
linuxPackages.digimend: Add PuercoPop as a maintainer

r-ryantm committed:
eksctl: 0.206.0 -> 0.207.0

Prince213 committed:
gitoxide: add patch removal note

r-ryantm committed:
terraform-providers.auth0: 1.14.0 -> 1.15.0

r-ryantm committed:
upbound-main: 0.39.0-0.rc.0.249.g7b07f31c -> 0.39.0-0.rc.0.301.gaa3d1f39

r-ryantm committed:
thunderbird-latest-unwrapped: 136.0.1 -> 137.0.1

amarshall committed:
librewolf-unwrapped: 136.0.4-1 -> 137.0-3

Previously reverted in 5fa303c382290270de12479432c7195d08b0a42a due to
an upstream issue, but that has now been resolved with `-3` release.

See also https://codeberg.org/librewolf/issues/issues/2400

jfly committed:
kikit: implement a workaround for non-deterministic `fetchFromGitHub`

See https://github.com/NixOS/nixpkgs/issues/84312 for details.

a-kenji committed:
nixos/all-tests: Fix typo

artemist committed:
freebsd: 14.1 → 14.2

GaetanLepage committed:
dolibarr: 21.0.0 -> 21.0.1

Diff: https://github.com/Dolibarr/dolibarr/compare/refs/tags/21.0.0...refs/tags/21.0.1

Changelog: https://github.com/Dolibarr/dolibarr/releases/tag/21.0.1

r-ryantm committed:
signal-cli: 0.13.13 -> 0.13.14

drupol commented on pkgs/development/python-modules/mkdocs-simple-blog/default.nix:
```diff
@@ -0,0 +1,55 @@
+  pillow,
+  ase,
+}:
+buildPythonPackage rec {
```
I think we should use the `finalAttrs` pattern here.

The `finalAttrs` pattern will let you remove the `rec` keyword ([see its implementation in Nix](https://github.com/NixOS/nix/blob/2d1d81114d72ace89ce08cd3bc93f4eb27a2975d/src/libexpr/nixexpr.cc#L121)).

Why is this pattern preferred to `rec` ?

Let's take this simple code example:

```nix
mkDerivation rec {
   foo = 1;
   bar = foo + 1;
}
```

and then `.overrideAttrs(old: { foo = 2; })`, you'll get `{ foo = 2; bar = 2; }` while with `finalAttrs` pattern, it would work correctly because it's a real fixed point.

Let me share a couple of useful links regarding the `finalAttrs` pattern:

1. History: https://discourse.nixos.org/t/avoid-rec-expresions-in-nixpkgs/8293
2. Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
3. Recent example of implementation: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

Feel free to reach out if you need some assistance.

Guelakais replied:
I know the contribution guide at least so far, so I know that you can only recommend it to me. I can also reject it if it is not essential for the pull request. Since I have already successfully submitted another python package using `rec`, I'll stick with it for now. I also find the links provided not very explanatory.

drupol replied:
Actually, I made a mistake, the finalAttrs pattern is not supported for buildPythonPackage builder.

drupol commented on pkgs/development/python-modules/mkdocs-simple-blog/default.nix:
```diff
@@ -0,0 +1,55 @@
+  version = "0.1.0";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.10";
```
This can be removed

Guelakais replied:
done

drupol commented on pkgs/development/python-modules/mkdocs-simple-blog/default.nix:
```diff
@@ -0,0 +1,55 @@
+
+  src = fetchFromGitHub {
+    owner = "FernandoCelmer";
+    repo = pname;
```
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21

Guelakais replied:
done

drupol commented on pkgs/development/python-modules/mkdocs-simple-blog/default.nix:
```diff
@@ -0,0 +1,55 @@
+  src = fetchFromGitHub {
+    owner = "FernandoCelmer";
+    repo = pname;
+    rev = "6a9971496ca8b7f53d0c78fe7859ca266583f866";
```
No tag on the repo?

Guelakais replied:
done

drupol commented on pkgs/development/python-modules/mkdocs-simple-blog/default.nix:
```diff
@@ -0,0 +1,55 @@
+    hash = "sha256-opl+8BRejr/cs3TrhFMZK9Xbq96kEQ6qVhI3EKDn858=";
+  };
+
+  propagatedBuildInputs = [
```
```suggestion
  dependencies = [
```

Guelakais replied:
done

drupol reviewed (changes requested):
Hello,

Have you had a look at the contributing guide?

There are rules on how to format the PR title and the commits, do you know anything about that?

I'll set the PR in draft, feel free to set it back ready for review when you think everything is OK, otherwise I'll close the PR in a month.

Thank you!

drupol commented:
FYI: `Mkdocs-simple-blog init at 0.1.0` is still wrong. Please read the contributing guide and have a look at pretty much 99% of all the pull requests in this project to give you some inspiration on how to properly write it.

r-ryantm committed:
terraform-providers.sops: 1.1.1 -> 1.2.0

r-ryantm committed:
terraform-providers.azuread: 3.2.0 -> 3.3.0

Guelakais commented:
That looks good now.

drupol reviewed (changes requested):
The commit message AND PR title are still incorrect.

Guelakais commented:
> The commit message AND PR title are still incorrect.

You're crying about a forgotten punctuation mark? Are you serious?

wegank committed:
python312Packages.jianpu-ly: 1.842 -> 1.844 (#394874)

wegank committed:
python312Packages.nodriver: 0.41 -> 0.43 (#394867)

wegank committed:
magic-vlsi: 8.3.522 -> 8.3.524 (#394599)

wegank committed:
astal.io: 0-unstable-2025-03-17 -> 0-unstable-2025-03-21 (#394480)

wegank committed:
apt: 2.9.33 -> 2.9.35 (#394587)

wolfgangwalther committed:
nixos/all-tests: Fix typo (#396622)

wolfgangwalther committed:
cockpit: 331 -> 336.2 (#387987)

wolfgangwalther committed:
gitoxide: add patch removal note (#396540)

wegank committed:
python312Packages.radish-bdd: 0.18.1 -> 0.18.2 (#395076)

wegank committed:
dnscrypt-proxy: 2.1.7 -> 2.1.8 (#393817)

wolfgangwalther committed:
linuxPackages.digimend: Add PuercoPop as a maintainer (#396494)

r-ryantm committed:
terraform-providers.incus: 0.3.0 -> 0.3.1

wolfgangwalther committed:
ansible: Migrate to by-name (#396163)

r-ryantm committed:
terraform-providers.spacelift: 1.20.4 -> 1.21.0

wegank committed:
linuxPackages.oci-seccomp-bpf-hook: 1.2.10 -> 1.2.11 (#394923)

wegank committed:
netmaker: 0.30.0 -> 0.90.0 (#393012)

wolfgangwalther committed:
cargo-deny: update description (#396246)

r-ryantm committed:
terraform-providers.newrelic: 3.59.0 -> 3.60.0

wolfgangwalther committed:
figlet: modernize package (#396078)

GaetanLepage committed:
dolibarr: 21.0.0 -> 21.0.1 (#396632)

wolfgangwalther committed:
nsq: Add blakesmith as maintainer (#395926)

wegank committed:
converseen: 0.12.2.5 -> 0.13.0.1 (#394790)

wegank committed:
klipper: 0.12.0-unstable-2025-03-12 -> 0.12.0-unstable-2025-03-25 (#393646)

Defelo committed:
uutils-findutils: 0.7.0 -> 0.8.0

gbtb committed:
nvtopPackages.full: 3.1.0 -> 3.2.0 (#395432)

wolfgangwalther committed:
eza: minor clean up (#395573)

wolfgangwalther committed:
plan-exporter: add update script (#395108)

wolfgangwalther committed:
yarn-berry: add meta.changelog (#394935)

wegank committed:
liblscp: 1.0.0 -> 1.0.1 (#393998)

wegank committed:
pipe-viewer: 0.5.4 -> 0.5.5 (#394418)

wegank committed:
libdatovka: 0.7.0 -> 0.7.1 (#394756)

wolfgangwalther committed:
espup: Set passthru.updateScript (#392535)

wegank committed:
sysdig-cli-scanner: 1.21.0 -> 1.22.0 (#393895)

wolfgangwalther committed:
guvcview: cleanup superfluous code (#392207)

wolfgangwalther committed:
kubernetes-helm: conditional completion generation, use finalAttrs (#391939)

artemist committed:
freebsd: 14.1 → 14.2 (#396624)

GaetanLepage committed:
fex: fix tests + minimize set of cloned submodules (#374694)

wegank committed:
gemmi: 0.7.0 -> 0.7.1 (#392932)

wolfgangwalther committed:
dozenal: drop (#394478)

wegank committed:
lima: 1.0.6 -> 1.0.7 (#394047)

wegank committed:
janet: 1.37.1 -> 1.38.0 (#392947)

nixpkgs-merge-bot[bot] committed:
rust-analyzer-unwrapped: 2025-03-24 -> 2025-03-31 (#396476)

Co-authored-by: oxalica <oxalica@users.noreply.github.com>

wegank committed:
rmg-wayland: 0.7.7 -> 0.7.8 (#393049)

wegank committed:
lxde.lxtask: 0.1.11 -> 0.1.12 (#396053)

trofi committed:
fzf: 0.61.0 -> 0.61.1

Changes: https://github.com/junegunn/fzf/releases/tag/v0.61.1

GaetanLepage committed:
azahar: fix SDL2 support

roberth committed:
nixops_unstablePlugins: put removed packages behind allowAliases (#395786)

booxter committed:
SDL_ttf: fix build on darwin

Without it, the build failed with:

Undefined symbols for architecture arm64:
  "_iconv", referenced from:
      _SDL_main in showfont.o
  "_iconv_close", referenced from:
      _SDL_main in showfont.o
  "_iconv_open", referenced from:
      _SDL_main in showfont.o

Signed-off-by: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>

r-ryantm committed:
nezha: 1.10.4 -> 1.10.8

maralorn committed:
nix-output-monitor: 2.1.5 -> 2.1.6

maralorn committed:
nix-output-monitor: Add formatting to update script

GaetanLepage committed:
azahar: fix SDL2 support (#396667)

Ericson2314 committed:
tbb_{2021_11,2022_0}: Use Ninja, build in parallel

These are just nice things to do where possible.

GaetanLepage committed:
uutils-findutils: 0.7.0 -> 0.8.0 (#396650)

Ericson2314 committed:
tbb_{2021_11,2022_0}: Fix build on Windows

fpletz committed:
librewolf-unwrapped: 136.0.4-1 -> 137.0-3 (#396571)

Commit:
vscodium: 1.99.02277 -> 1.99.02289

r-ryantm committed:
pomerium-cli: 0.29.0 -> 0.29.1

r-ryantm committed:
signalbackup-tools: 20250331-1 -> 20250406-1

pbsds committed:
nixos/kafka: Added cluster/MirrorMaker testcase (#317496)

r-ryantm committed:
cargo-temp: 0.3.2 -> 0.3.3

Ericson2314 committed:
tbb_{2021_11,2022_0}: Fix build on Windows (#396676)

lukegb committed:
pomerium-cli: 0.29.0 -> 0.29.1 (#396691)

dotlambda committed:
python313Packages.syrupy: 4.9.0 -> 4.9.1 (#394149)

Commit:
vscode-language-packs: 1.98.2025031209 -> 1.99.2025040209

nixpkgs-merge-bot[bot] committed:
eksctl: 0.206.0 -> 0.207.0 (#396497)

Co-authored-by: xrelkd <xrelkd@users.noreply.github.com>

nixpkgs-merge-bot[bot] committed:
theforceengine: 1.22.200 -> 1.22.300 (#396005)

Co-authored-by: devusb <devusb@users.noreply.github.com>

Ericson2314 committed:
nixVersions.nix_2_28: init at 2.28.1pre (#396710)

Aleksanaa committed:
sunshine: fix /bin/sleep in systemd service (#395760)

r-ryantm committed:
httpyac: 6.16.6 -> 6.16.7

r-ryantm committed:
cryptomator: 1.15.1 -> 1.15.2

mkg20001 committed:
qlog: 0.42.2 -> 0.43.0 (#396046)

drupol committed:
vscode-language-packs: 1.98.2025031209 -> 1.99.2025040209 (#396709)

drupol committed:
nix-output-monitor: 2.1.5 -> 2.1.6 (#396673)

drupol committed:
signal-cli: 0.13.13 -> 0.13.14 (#396635)

drupol committed:
signalbackup-tools: 20250331-1 -> 20250406-1 (#396692)

drupol committed:
httpyac: 6.16.6 -> 6.16.7 (#396726)

nixpkgs-merge-bot[bot] committed:
upbound-main: 0.39.0-0.rc.0.249.g7b07f31c -> 0.39.0-0.rc.0.301.gaa3d1f39 (#396552)

Co-authored-by: jljox <jljox@users.noreply.github.com>

r-ryantm committed:
pkgsite: 0-unstable-2025-03-21 -> 0-unstable-2025-04-01

flokli committed:
signal-desktop-source: add update script; 7.48.0 -> 7.49.0 (#395596)

vcunat committed:
kikit: 1.7.1 -> 1.7.2, python3Packages.pcbnewtransition: 0.5.0 -> 0.5.2: Fix kikit build (#395213)

drupol committed:
open-webui: 0.6.1 -> 0.6.2

changelog: https://github.com/open-webui/open-webui/releases/tag/v0.6.2
diff: https://github.com/open-webui/open-webui/compare/v0.6.1..v0.6.2

jian-lin committed:
emacs.pkgs.hyperdrive: 20250331.427 -> 20250406.2225

Fix https://hydra.nix-community.org/build/4641165

nixpkgs-merge-bot[bot] committed:
pkgsite: 0-unstable-2025-03-21 -> 0-unstable-2025-04-01 (#396735)

Co-authored-by: phanirithvij <madlad@duck.com>

drupol committed:
treewide: remove unneeded string interpolation

RossComputerGuy committed:
mattermostLatest: 10.6.0 -> 10.6.1 (#392845)

drupol committed:
vscodium: 1.99.02277 -> 1.99.02289 (#396685)

nixpkgs-merge-bot[bot] committed:
nezha: 1.10.4 -> 1.10.8 (#396671)

Co-authored-by: Moraxyc <Moraxyc@users.noreply.github.com>

GaetanLepage committed:
treewide: remove unneeded string interpolation (#396740)

nixpkgs-merge-bot[bot] committed:
cryptomator: 1.15.1 -> 1.15.2 (#396727)

Co-authored-by: bachp <bachp@users.noreply.github.com>

jian-lin committed:
emacs.pkgs.hyperdrive: 20250331.427 -> 20250406.2225 (#396739)

yuyuyureka committed:
gclient2nix: do not depend on nixpkgs path

GaetanLepage committed:
open-webui: 0.6.1 -> 0.6.2 (#396738)

nixpkgs-merge-bot[bot] committed:
cargo-temp: 0.3.2 -> 0.3.3 (#396700)

Co-authored-by: matthiasbeyer <mail@beyermatthias.de>

wolfgangwalther committed:
rke2: use `finalAttrs` to avoid `let` and `rec` bindings (#393720)

K900 committed:
gclient2nix: do not depend on nixpkgs path (#396741)

RossComputerGuy committed:
selinux-refpolicy: init 2.20250213 (#396155)

nixpkgs-merge-bot[bot] committed:
ansible-navigator: 25.1.0 -> 25.4.0 (#396466)

Co-authored-by: Melkor333 <Melkor333@users.noreply.github.com>

vcunat committed:
thunderbird: 136.0.1 -> 137.0.1 (#396572)

GaetanLepage committed:
gitlab-runner: Add main program, version check hook, and Nix update script. Fix Darwin builds. (#395906)

wegank committed:
SDL_ttf: fix build on darwin (#396668)

Guelakais committed:
Update pkgs/development/python-modules/mkdocs-simple-blog/default.nix

Co-authored-by: Gaétan Lepage <33058747+GaetanLepage@users.noreply.github.com>

Guelakais committed:
Update pkgs/development/python-modules/mkdocs-simple-blog/default.nix

Co-authored-by: Gaétan Lepage <33058747+GaetanLepage@users.noreply.github.com>

Guelakais committed:
Update pkgs/development/python-modules/mkdocs-simple-blog/default.nix

Co-authored-by: Gaétan Lepage <33058747+GaetanLepage@users.noreply.github.com>

Guelakais committed:
Merge remote-tracking branch 'origin/mkdocs-simple-blog' into mkdocs-simple-blog

drupol commented:
It looks like you accidentally mass-pinged a bunch of people, which are now subscribed
and getting notifications for everything in this pull request. Unfortunately, they
cannot be automatically unsubscribed from the issue (removing review request does not
unsubscribe), therefore development cannot continue in this pull request anymore.

Please create a new pull request, link back to this one and ping the
people actually involved in here over there. For the next time, remember to set your PR to draft status before rebasing. In draft status, you can preview the list of maintainers that are about to be requested for review, which allows you to sidestep this issue.

In order to avoid this in the future, there are instructions for how to properly
rebase between branches in our [contribution guidelines](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#rebasing-between-branches-ie-from-master-to-staging).
Setting your pull request to draft prior to rebasing is strongly recommended.
In draft status, you can preview the list of people that are about to be requested
for review, which allows you to sidestep this issue.
This is not a bulletproof method, though, as OfBorg still does review requests even on draft PRs.
